lib/heroku/command/pg.rb

```diff
@@ -85,16 +85,29 @@ def promote
     end
     validate_arguments!
 
+    db = db.sub(/_URL$/, '') # allow promoting with a var name
     addon = resolve_addon!(db)
 
-    attachment_name = 'DATABASE'
-    action "Promoting #{addon['name']} to #{attachment_name}_URL on #{app}" do
+    promoted_name = 'DATABASE'
+
+    action "Ensuring an alternate alias for existing #{promoted_name}" do
+      backup = find_or_create_non_database_attachment(app)
+
+      if backup
+        @status = backup['name']
+      else
+        @status = "not needed"
```

Comment:
It suffixes the `...done` message for the action. It used to replace it, but now it'd look like:

```
Ensuring an alternate alias for existing DATABASE... done, HEROKU_POSTGRESQL_AQUA
```